dualis_connector/dualis_service.py

```diff
@@ -1,125 +1,138 @@
-import logging
```

Comment:
Could you please check where this sudden big change set comes from? I.e. is maybe the text encoding or line endings off?

Reply:
I commited via WSL (Ubuntu) so maybe it auto converted line endings to LF

Reply:
That sounds like a valid root cause, yes.
Could you please have another look and ensure there is a clean diff for the actually affected parts only? Than we can merge this PR.

Reply:
done